2019.08 release notes (2019-10-01).MD

```diff
@@ -0,0 +1,35 @@
+# 2019.08 release notes (2019-10-01)
+Packages released:
+- @patternfly/patternfly@2.x
```

Comment:
lgtm other than the version. @dgutride just to confirm, is this `2.33.5`?

Reply:
yep - that's the one we pulled into react-core